common/scala/src/main/scala/whisk/core/entity/WhiskStore.scala

```diff
@@ -205,12 +208,13 @@ object WhiskEntityQueries {
     def listEntitiesInNamespace[A <: WhiskEntity](
         db: ArtifactStore[A],
         namespace: EntityPath,
-        includeDocs: Boolean)(
+        includeDocs: Boolean,
+        stale: StaleParameter = StaleParameter.No)(
             implicit transid: TransactionId): Future[Map[String, List[JsObject]]] = {
         implicit val ec = db.executionContext
         val startKey = List(namespace.toString)
         val endKey = List(namespace.toString, TOP)
-        db.query(viewname(ENTITIES), startKey, endKey, 0, 0, includeDocs, descending = true, reduce = false) map {
+        db.query(viewname(ENTITIES), startKey, endKey, 0, 0, includeDocs, descending = true, reduce = false, stale = stale) map {
```

Comment:
I would have used just stale instead of stale = stale; since the name is indicative vs just a Boolean.